### ISSUE TITLE
Search collection helper result

### ISSUE THREAD

**Issue by brylie** [yogiben/meteor-admin#212]
When displaying the data table from any collection, the search field only seems to operate on visible fields from the current collection. I.e. any data displayed from collection helpers may not be searched, even though it may be visible in the data table.

Implement search capabilities for collection helper data within a data table.

**Comment by mpowaga** [comment]
Virtual fields cannot be searched through the database. One possible solutions is to provide a hook for `changeSelector` [callback](https://github.com/yogiben/meteor-admin/blob/master/lib/both/startup.coffee#L34).

**Comment by brylie** [comment]
Alright, I am a bit unclear on how to use the changeSelector callback.

**Comment by mordra** [comment]
Yeah, I'm in the same boat, would love some idiot-proof example of how to do this after following that link and feeling inadequate...

**Comment by mpowaga** [comment]
I wasn't clear enough. I meant that it should be possible to define custom `changeSelector` callback. But that's not possible yet.

**Comment by brylie** [comment]
OK, thanks for the clarification. This is a really important feature for a [project](https://github.com/brylie/juhani-wellbeing) that I am working on. We have a [collection](https://github.com/brylie/juhani-wellbeing/blob/master/both/collections/activities.js#L8) uses three collection helpers, linking to and/or formatting data that are of primary importance. When users view the submissions for the primary collection, they will reflexively use the search field expecting to filter for results from the relationship collection helpers.

**Comment by mordra** [comment]
+1, [Column](Not searchable) isn't going to go down well with users hehe

**Comment by brylie** [comment]
There is an [open issue on the aldeed:tabular package](https://github.com/aldeed/meteor-tabular/issues/151) that is related to this issue.

**Comment by niranjans** [comment]
+1. Anyone got any type of workaround for this yet?

**Comment by webmagnets** [comment]
+1
